qemu/tests/cfg/in_place_upgrade_legacy.cfg

```diff
@@ -0,0 +1,62 @@
+- in_place_upgrade_legacy:
+    only RHEL.7.9
+    type = in_place_upgrade_legacy
+    pre_release = "release 7"
+    post_release = "release 8"
+    new_kernel_ver = "4.18"
+    login_timeout = 300
+    release_check = "cat /etc/redhat-release"
+    fix_permit = "echo "PermitRootLogin yes" >> /etc/ssh/sshd_config "
+    check_kernel = "uname -r"
+    yum_update = "yum -y update"
+    yum_utils = "yum install yum-utils -y"
```

Comment:
It can be installed via `utils_package`.

Reply:
will update, thanks